Stanford/Utility/Install/CreateMenuLinks.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * This class creates new menu links.
+ */
+
+namespace Stanford\Utility\Install;
+
+class CreateMenuLinks extends \AbstractInstallTask {
+
+
+  public function execute(&$args = array()) {
+    /**
```

Comment:
An unfortunate side effect of having multiple branches and pull requests in flight.